### ISSUE TITLE
Appium Inspector Session Expired in less than 60 seconds

### ISSUE THREAD

**Issue by pavelbaramov** [appium/appium-desktop#525]
## Appium Desktop

- [ ] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

Appium Inspector Session Expired in less than 60 seconds after the first start of the Inspector.
I tried to set in Desired Capabilities a variable **newCommandTimeout** to **600** (in seconds) but the issues remains.

_Error message appears into the Inspector:_
```Error 
Session closed due to inactivity
```

## Environment

- I am running Appium Desktop version 1.6 and Appium server 1.8;
- I am on:
    -  Mac

## Request

How could I set this session timeout to be more than 60sec?

```[MJSONWP] Calling AppiumDriver.getSession() with args: ["7258ce6b-0243-4c95-b933-d800dce94eec"]
[XCUITest] Executing command 'getSession'
[JSONWP Proxy] Proxying [GET /] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n    \"capabilities\" : {\n      \"device\" : \"iphone\",\n      \"browserName\" : \"hc-helios\",\n      \"sdkVersion\" : \"11.3\",\n      \"CFBundleIdentifier\" : \"com.devolo.hc-helios\"\n    }\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[XCUITest] Merging WDA caps over Appium caps for session detail response
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /window/size] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/window/size] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"width\" : 375,\n    \"height\" : 667\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[MJSONWP] Responding to client with driver.getSession() result: {"udid":"094D2F40-9B61-4E66-93ED-D53CF948759E","app":"/Users/pb/Library/Developer/Xcode/DerivedData/hc-helios-cjlmjdyhjnjmmpfsmdhaaglvpaoo/Build/Products/Debug-iphonesimulator/hc-helios.app","automationName":"XCUITest","deviceName":"iPhone 7","newCommandTimeout":600,"noReset":true,"platformName":"iOS","showXcodeLog":true,"connectHardwareKeyboard":true,"device":"iphone","browserName":"hc-helios","sdkVersion":"11.3","CFBundleIdentifier":"com.devolo.hc-helios","pixelRatio":2,"statBarHeight":20,"viewportRect":{"left":0,"top":40,"width":750,"height":1294}}
[HTTP] <-- GET /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec 200 539 ms - 629
[HTTP] 
[HTTP] --> GET /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec
[HTTP] {}
[MJSONWP] Calling AppiumDriver.getSession() with args: ["7258ce6b-0243-4c95-b933-d800dce94eec"]
[XCUITest] Executing command 'getSession'
[JSONWP Proxy] Proxying [GET /] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n    \"capabilities\" : {\n      \"device\" : \"iphone\",\n      \"browserName\" : \"hc-helios\",\n      \"sdkVersion\" : \"11.3\",\n      \"CFBundleIdentifier\" : \"com.devolo.hc-helios\"\n    }\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[XCUITest] Merging WDA caps over Appium caps for session detail response
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /window/size] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/window/size] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"width\" : 375,\n    \"height\" : 667\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[MJSONWP] Responding to client with driver.getSession() result: {"udid":"094D2F40-9B61-4E66-93ED-D53CF948759E","app":"/Users/pb/Library/Developer/Xcode/DerivedData/hc-helios-cjlmjdyhjnjmmpfsmdhaaglvpaoo/Build/Products/Debug-iphonesimulator/hc-helios.app","automationName":"XCUITest","deviceName":"iPhone 7","newCommandTimeout":600,"noReset":true,"platformName":"iOS","showXcodeLog":true,"connectHardwareKeyboard":true,"device":"iphone","browserName":"hc-helios","sdkVersion":"11.3","CFBundleIdentifier":"com.devolo.hc-helios","pixelRatio":2,"statBarHeight":20,"viewportRect":{"left":0,"top":40,"width":750,"height":1294}}
[HTTP] <-- GET /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec 200 349 ms - 629
[HTTP] 
[HTTP] --> GET /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec
[HTTP] {}
[MJSONWP] Calling AppiumDriver.getSession() with args: ["7258ce6b-0243-4c95-b933-d800dce94eec"]
[XCUITest] Executing command 'getSession'
[JSONWP Proxy] Proxying [GET /] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n    \"capabilities\" : {\n      \"device\" : \"iphone\",\n      \"browserName\" : \"hc-helios\",\n      \"sdkVersion\" : \"11.3\",\n      \"CFBundleIdentifier\" : \"com.devolo.hc-helios\"\n    }\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[XCUITest] Merging WDA caps over Appium caps for session detail response
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[Xcode] 04000000-0000-0000-0598-000000000000", "05000000-0000-0000-0598-000000000000"} AND (1 == 1 OR identifier == 0 OR frame == 0 OR value == 0 OR title == 0 OR label == 0 OR elementType == 0 OR enabled == 0 OR placeholderValue == 0)'
[Xcode]     t =   174.61s Find: Identity Binding
[Xcode]     t =   174.61s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   174.64s Find: Descendants matching type Window
[Xcode]     t =   174.64s Find: Elements matching predicate 'wdUID IN {"04000000-0000-0000-0598-000000000000", "05000000-0000-0000-0598-000000000000"} AND (1 == 1 OR identifier == 0 OR frame == 0 OR value == 0 OR title == 0 OR label == 0 OR elementType == 0 OR enabled == 0 OR placeholderValue == 0)'
[Xcode]     t =   174.64s Find: Identity Binding
[Xcode]     t =   174.64s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   174.64s Find: Descendants matching type Window
[Xcode]     t =   174.64s Find: Elements matching predicate 'wdUID IN {"04000000-0000-0000-0598-000000000000", "05000000-0000-0000-0598-000000000000"} AND (1 == 1 OR identifier == 0 OR frame == 0 OR value == 0 OR title == 0 OR label == 0 OR elementType == 0 OR enabled == 0 OR placeholderValue == 0)'
[Xcode]     t =   174.64s Find: Identity Binding
[Xcode]     t =   174.64s Find the Window
[Xcode]     t =   174.64s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   174.67s     Find: Descendants matching type Window
[Xcode]     t =   174.67s     Find: Elements matching predicate 'wdUID IN {"04000000-0000-0000-0598-000000000000", "05000000-0000-0000-0598-000000000000"} AND (1 == 1 OR identifier == 0 OR frame == 0 OR value == 0 OR title == 0 OR label == 0 OR elementType == 0 OR enabled == 0 OR placeholderValue == 0)'
[Xcode]     t =   174.67s     Find: Identity Binding
[Xcode] 2018-05-08 11:04:45.248379+0300 WebDriverAgentRunner-Runner[38880:2585081] Using singleton test manager
[Xcode]     t =   175.79s Find the Window
[Xcode]     t =   175.79s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   175.82s     Find: Descendants matching type Window
[Xcode]     t =   175.82s     Find: Elements matching predicate 'wdUID IN {"04000000-0000-0000-0598-000000000000", "05000000-0000-0000-0598-000000000000"} AND (1 == 1 OR identifier == 0 OR frame == 0 OR value == 0 OR title == 0 OR label == 0 OR elementType == 0 OR enabled == 0 OR placeholderValue == 0)'
[Xcode]     t =   175.82s     Find: Identity Binding
[Xcode] 2018-05-08 11:04:46.398730+0300 WebDriverAgentRunner-Runner[38880:2585081] Using singleton test manager
[Xcode]     t =   175.87s Find the Application "com.devolo.hc-helios" 0x6040002c1490
[Xcode]     t =   175.87s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   175.94s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   187.81s Find the Application "com.devolo.hc-helios" 0x6040002c1490
[Xcode]     t =   187.81s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   187.84s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   187.86s Find: Descendants matching type StatusBar
[Xcode]     t =   187.86s Get all elements bound by accessibility element for: Descendants matching type StatusBar
[Xcode]     t =   187.86s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   187.88s     Find: Descendants matching type StatusBar
[Xcode]     t =   187.89s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   187.92s Find: Descendants matching type StatusBar
[Xcode]     t =   187.92s Find: Identity Binding
[Xcode]     t =   187.94s Find the StatusBar
[Xcode]     t =   187.94s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   187.96s     Find: Descendants matching type StatusBar
[Xcode]     t =   187.96s     Find: Identity Binding
[Xcode]     t =   188.04s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   188.06s Find: Descendants matching type StatusBar
[Xcode]     t =   188.06s Get all elements bound by accessibility element for: Descendants matching type StatusBar
[Xcode]     t =   188.06s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   188.08s     Find: Descendants matching type StatusBar
[Xcode]     t =   188.09s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   188.12s Find: Descendants matching type StatusBar
[Xcode]     t =   188.13s Find: Identity Binding
[Xcode]     t =   188.14s Find the StatusBar
[Xcode]     t =   188.14s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   188.16s     Find: Descendants matching type StatusBar
[Xcode]     t =   188.16s     Find: Identity Binding
[Xcode]     t =   188.17s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   188.20s Find: Descendants matching type StatusBar
[Xcode]     t =   188.20s Get all elements bound by accessibility element for: Descendants matching type StatusBar
[Xcode]     t =   188.20s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   188.22s     Find: Descendants matching type StatusBar
[Xcode]     t =   188.22s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   188.26s Find: Descendants matching type StatusBar
[Xcode]     t =   188.26s Find: Identity Binding
[Xcode]     t =   188.27s Find the StatusBar
[Xcode]     t =   188.27s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   188.29s     Find: Descendants matching type StatusBar
[Xcode]     t =   188.29s     Find: Identity Binding
[Xcode]     t =   188.30s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   202.82s Find the Application "com.devolo.hc-helios" 0x6040002c1490
[Xcode]     t =   202.82s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   202.84s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   202.86s Find: Descendants matching type StatusBar
[Xcode]     t =   202.86s Get all elements bound by accessibility element for: Descendants matching type StatusBar
[Xcode]     t =   202.87s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   202.89s     Find: Descendants matching type StatusBar
[Xcode]     t =   202.89s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   202.93s Find: Descendants matching type StatusBar
[Xcode]     t =   202.93s Find: Identity Binding
[Xcode]     t =   202.94s Find the StatusBar
[Xcode]     t =   202.94s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   202.96s     Find: Descendants matching type StatusBar
[Xcode]     t =   202.96s     Find: Identity Binding
[Xcode]     t =   202.98s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   203.00s Find: Descendants matching type StatusBar
[Xcode]     t =   203.00s Get all elements bound by accessibility element for: Descendants matching type StatusBar
[Xcode]     t =   203.00s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   203.02s     Find: Descendants matching type StatusBar
[Xcode]     t =   203.03s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   203.06s Find: Descendants matching type StatusBar
[Xcode]     t =   203.06s Find: Identity Binding
[Xcode]     t =   203.08s Find the StatusBar
[Xcode]     t =   203.08s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   203.10s     Find: Descendants matching type StatusBar
[Xcode]     t =   203.10s     Find: Identity Binding
[Xcode]     t =   203.11s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   217.82s Find the Application "com.devolo.hc-helios" 0x6040002c1490
[Xcode]     t =   217.82s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   217.85s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   217.87s Find: Descendants matching type StatusBar
[Xcode]     t =   217.87s Get all elements bound by accessibility element for: Descendants matching type StatusBar
[Xcode]     t =   217.87s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   217.89s     Find: Descendants matching type StatusBar
[Xcode]     t =   217.90s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   217.93s Find: Descendants matching type StatusBar
[Xcode]     t =   217.93s Find: Identity Binding
[Xcode]     t =   217.94s Find the StatusBar
[Xcode]     t =   217.94s     Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   217.97s     Find: Descendants matching type StatusBar
[Xcode]     t =   217.97s     Find: Identity Binding
[Xcode]     t =   217.98s Snapshot accessibility hierarchy for app with pid 38917
[Xcode]     t =   218.00s Find: Descendants matching type StatusBar
[Xcode]     t =   218.00s Get all elements bound by accessibility element for: Descendants mat
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /window/size] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/window/size] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"width\" : 375,\n    \"height\" : 667\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[MJSONWP] Responding to client with driver.getSession() result: {"udid":"094D2F40-9B61-4E66-93ED-D53CF948759E","app":"/Users/pb/Library/Developer/Xcode/DerivedData/hc-helios-cjlmjdyhjnjmmpfsmdhaaglvpaoo/Build/Products/Debug-iphonesimulator/hc-helios.app","automationName":"XCUITest","deviceName":"iPhone 7","newCommandTimeout":600,"noReset":true,"platformName":"iOS","showXcodeLog":true,"connectHardwareKeyboard":true,"device":"iphone","browserName":"hc-helios","sdkVersion":"11.3","CFBundleIdentifier":"com.devolo.hc-helios","pixelRatio":2,"statBarHeight":20,"viewportRect":{"left":0,"top":40,"width":750,"height":1294}}
[HTTP] <-- GET /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec 200 338 ms - 629
[HTTP] 
[HTTP] --> GET /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec
[HTTP] {}
[MJSONWP] Calling AppiumDriver.getSession() with args: ["7258ce6b-0243-4c95-b933-d800dce94eec"]
[XCUITest] Executing command 'getSession'
[JSONWP Proxy] Proxying [GET /] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n    \"capabilities\" : {\n      \"device\" : \"iphone\",\n      \"browserName\" : \"hc-helios\",\n      \"sdkVersion\" : \"11.3\",\n      \"CFBundleIdentifier\" : \"com.devolo.hc-helios\"\n    }\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[XCUITest] Merging WDA caps over Appium caps for session detail response
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /wda/screen] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/wda/screen] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"statusBarSize\" : {\n      \"width\" : 375,\n      \"height\" : 20\n    },\n    \"scale\" : 2\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[JSONWP Proxy] Proxying [GET /window/size] to [GET http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E/window/size] with no body
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"width\" : 375,\n    \"height\" : 667\n  },\n  \"sessionId\" : \"9032C624-E9E3-435C-B76C-56AE785D5D8E\",\n  \"status\" : 0\n}"
[MJSONWP] Responding to client with driver.getSession() result: {"udid":"094D2F40-9B61-4E66-93ED-D53CF948759E","app":"/Users/pb/Library/Developer/Xcode/DerivedData/hc-helios-cjlmjdyhjnjmmpfsmdhaaglvpaoo/Build/Products/Debug-iphonesimulator/hc-helios.app","automationName":"XCUITest","deviceName":"iPhone 7","newCommandTimeout":600,"noReset":true,"platformName":"iOS","showXcodeLog":true,"connectHardwareKeyboard":true,"device":"iphone","browserName":"hc-helios","sdkVersion":"11.3","CFBundleIdentifier":"com.devolo.hc-helios","pixelRatio":2,"statBarHeight":20,"viewportRect":{"left":0,"top":40,"width":750,"height":1294}}
[HTTP] <-- GET /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec 200 329 ms - 629
[HTTP] 
[HTTP] --> DELETE /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec
[HTTP] {}
[MJSONWP] Calling AppiumDriver.deleteSession() with args: ["7258ce6b-0243-4c95-b933-d800dce94eec"]
[BaseDriver] Event 'quitSessionRequested' logged at 1525766759302 (11:05:59 GMT+0300 (EEST))
[Appium] Removing session 7258ce6b-0243-4c95-b933-d800dce94eec from our master session list
[JSONWP Proxy] Proxying [DELETE /session/7258ce6b-0243-4c95-b933-d800dce94eec] to [DELETE http://localhost:8100/session/9032C624-E9E3-435C-B76C-56AE785D5D8E] with no body
[HTTP] --> GET /wd/hub/sessions
[HTTP] {}
[MJSONWP] Calling AppiumDriver.getSessions() with args: []
[MJSONWP] Responding to client with driver.getSessions() result: []
[HTTP] <-- GET /wd/hub/sessions 200 1 ms - 40
[HTTP] 
[JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n\n  },\n  \"sessionId\" : \"E3524420-91A3-499D-BF2F-4F3F5C692706\",\n  \"status\" : 0\n}"
[XCUITest] Not clearing log files. Use `clearSystemFiles` capability to turn on.
[iOSLog] Stopping iOS log capture
[BaseDriver] Event 'quitSessionFinished' logged at 1525766759522 (11:05:59 GMT+0300 (EEST))
[MJSONWP] Received response: null
[MJSONWP] But deleting session, so not returning
[MJSONWP] Responding to client with driver.deleteSession() result: null
[HTTP] <-- DELETE /wd/hub/session/7258ce6b-0243-4c95-b933-d800dce94eec 200 221 ms - 76
[HTTP] ```

**Comment by pilipala195** [comment]
Same question here, any help?

**Comment by MoOoG** [comment]
I have the same problem - set Desired Capability 

- "newCommandTimeout": 900
and
- "newCommandTimeout": "900"

but session closes anyways in the same amount of time.

**Comment by dpgraham** [comment]
This shouldn't be happening so quickly

I'm going to put in a fix for this. For now, I would revert to previous version.

**Comment by dpgraham** [comment]
This is fixed in 1.6.1
